File: rust_internal_api_system/src/grpc_key_string_maps/room_control_service.rs
pub struct CreateRoomRequest {
}
                    
impl CreateRoomRequest {
    pub const roomName: &str = "roomName";
}
                    


pub struct CreateRoomResponse {
}
                    
impl CreateRoomResponse {
    pub const success: &str = "success";
    pub const error: &str = "error";
}
                    


pub struct AllowJoinRequest {
}
                    
impl AllowJoinRequest {
    pub const roomName: &str = "roomName";
    pub const identity: &str = "identity";
}
                    


pub struct AllowJoinResponse {
}
                    
impl AllowJoinResponse {
    pub const accessToken: &str = "accessToken";
    pub const error: &str = "error";
}
                    


pub struct RoomInfo {
}
                    
impl RoomInfo {
    pub const roomName: &str = "roomName";
    pub const numberOfParticipants: &str = "numberOfParticipants";
}
                    


pub struct ListRoomsResponse {
}
                    
impl ListRoomsResponse {
    pub const rooms: &str = "rooms";
    pub const error: &str = "error";
}
                    


pub struct DeleteRoomRequest {
}
                    
impl DeleteRoomRequest {
    pub const roomName: &str = "roomName";
}
                    


pub struct DeleteRoomResponse {
}
                    
impl DeleteRoomResponse {
    pub const success: &str = "success";
    pub const error: &str = "error";
}

File: java_free_map_backend_system/app/src/main/kotlin/grpc_key_string_maps/room_control_service.kt

package grpc_key_string_maps

class room_control_service_key_string_maps {
    companion object {
            class CreateRoomRequest {
                companion object {
                    @JvmField 
                    var roomName: String = "roomName"
                    val __property_list__: List<String> = listOf("roomName")
                }
            }

            class CreateRoomResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var success: String = "success"
                    val __property_list__: List<String> = listOf("error", "success")
                }
            }

            class AllowJoinRequest {
                companion object {
                    @JvmField 
                    var roomName: String = "roomName"
                    var identity: String = "identity"
                    val __property_list__: List<String> = listOf("roomName", "identity")
                }
            }

            class AllowJoinResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var accessToken: String = "accessToken"
                    val __property_list__: List<String> = listOf("error", "accessToken")
                }
            }

            class RoomInfo {
                companion object {
                    @JvmField 
                    var roomName: String = "roomName"
                    var numberOfParticipants: String = "numberOfParticipants"
                    val __property_list__: List<String> = listOf("roomName", "numberOfParticipants")
                }
            }

            class ListRoomsResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var rooms: String = "rooms"
                    val __property_list__: List<String> = listOf("error", "rooms")
                }
            }

            class DeleteRoomRequest {
                companion object {
                    @JvmField 
                    var roomName: String = "roomName"
                    val __property_list__: List<String> = listOf("roomName")
                }
            }

            class DeleteRoomResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var success: String = "success"
                    val __property_list__: List<String> = listOf("error", "success")
                }
            }
    }
}